configure.ac

```diff
@@ -36,7 +36,7 @@ AX_C_BIGENDIAN_CROSS
 AC_FUNC_ALLOCA
 AC_CHECK_HEADERS([errno.h arpa/inet.h fcntl.h limits.h locale.h malloc.h stddef.h sys/file.h sys/mount.h sys/param.h sys/statvfs.h syslog.h wchar.h])
 AC_CHECK_HEADERS([execinfo.h ucontext.h sched.h])
-AC_CHECK_HEADERS([sys/xattr.h])
+AC_CHECK_HEADERS([sys/sysmacros.h sys/xattr.h])
```

Comment:
Keep sys/xattr.h on its own line and add a new AC_CHECK_HEADERS entry for sys/sysmacros.h